docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.9"
+
+services:
+  mongodb:
+    image: mongo
```

Comment:
Wouldn't be better to pin a version to avoid breaking changes? Then we can also have control over the dev environment  and update it according to the new versions

Reply:
For sure!